packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -225,7 +255,7 @@ const AppNavigation = ({ items, primaryContent }: Props) => {
                                             </IconWrapper>
                                         )}
 
-                                        <Text>{title}</Text>
+                                        <TextHover>{title}</TextHover>
```

Comment:
![Screenshot 2021-06-21 at 12 08 19](https://user-images.githubusercontent.com/33235762/122745635-7ded7a80-d289-11eb-88b4-3966420ddc19.png)

Probably also the icon should be in the shadow? Btw padding of the shadow should be much bigger than this.

Reply:
94822a6